bin/ci-test

```diff
@@ -47,15 +47,16 @@ function build() {
 
   set -x
   docker build \
-    --build-arg TARGETARCH \
+    --build-arg TARGETARCH="$TARGETARCH" \
```

Comment:
This reverted part of https://github.com/dependabot/dependabot-core/commit/89df66560c3192871d3c41a34a2fa151a4fef674, was it intentional?

Reply:
Oh sorry, I thought I had typo'd that. I *think* I might have been getting a build error with it on my ARM system. I'm happy to try again in a moment.

Reply:
No problem, it's a very unknown docker feature, I didn't know it until @jeffwidman introduced it. I can see how you thought you typo'ed it!

Reply:
Confirmed it doesn't work with just `--build-arg TARGETARCH`.

Reply:
Follow up PR: https://github.com/dependabot/dependabot-core/pull/6549